File: app/src/test/java/com/mgaetan89/showsrage/fragment/SettingsDisplayFragmentTest.java
package com.mgaetan89.showsrage.fragment;

import com.mgaetan89.showsrage.R;

import org.junit.Test;

import static org.assertj.core.api.Assertions.assertThat;

public class SettingsDisplayFragmentTest {
	@Test
	public void getTitleResourceId() {
		assertThat(new SettingsDisplayFragment().getTitleResourceId()).isEqualTo(R.string.display);
	}

	@Test
	public void getXmlResourceFile() {
		assertThat(new SettingsDisplayFragment().getXmlResourceFile()).isEqualTo(R.xml.settings_display);
	}
}

File: app/src/test/kotlin/com/mgaetan89/showsrage/fragment/SettingsDisplayFragmentTest.kt
package com.mgaetan89.showsrage.fragment

import com.mgaetan89.showsrage.R
import org.assertj.core.api.Assertions.assertThat
import org.junit.Test

class SettingsDisplayFragmentTest {
	@Test
	fun getTitleResourceId() {
		assertThat(SettingsDisplayFragment.newInstance().getTitleResourceId()).isEqualTo(R.string.display)
	}

	@Test
	fun getXmlResourceFile() {
		assertThat(SettingsDisplayFragment.newInstance().getXmlResourceFile()).isEqualTo(R.xml.settings_display)
	}
}
